spec/Methods.tex

```diff
@@ -163,6 +162,12 @@ \section{The Method Receiver and the {\em this} Argument}
 methods}.  Methods may also be defined to have \chpl{type}
 receivers---these are known as \emph{type methods}.
 
+Note that within a method for a class \chpl{C}, the type of \chpl{this}
+is generally \chpl{borrowed C}. Within a type method on a class \chpl{C},
+\chpl{this} refers to a class type with management and nilability matching
```

Comment:
add `C` : `refers to the class type \chpl{C}`

Reply:
I'm not following?

Reply:
Oh, I got it.